show_notes/Episode_18_Part2.md

```diff
@@ -0,0 +1,20 @@
+# Episode 18, Part 2: Android Everywhere
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/92)
+
+In the second part of episode #18 Artur, Hannes and Artem discuss about Android TV, Chromecast and Android Auto.
```

Comment:
_discuss Android TV, Chromecast and Android Auto_

Reply:
What do you mean?

Reply:
No `about`

Reply:
Yep, thanks @vanniktech.